src/components/Header.tsx

```diff
@@ -2,28 +2,44 @@ import React from 'react';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
 import MenuIcon from '@material-ui/icons/Menu';
 import { Link } from 'react-router-dom';
+import styled from 'styled-components';
 
-const onMenuClick = (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
-  //handle when the menu button is clicked here
-};
+import Logo from './Logo';
+import { Container } from '../App.styles';
+
+const headerVerticalMargin = 24;
 
-function Header() {
+const StyledHeader = styled(AppBar)`
+  && {
+    background: none;
+    box-shadow: none;
+    margin-top: ${headerVerticalMargin}px;
```

Comment:
super duper nit: sort alphabetically

Reply:
You mean all CSS properties sorted alphabetically?

Reply:
yeh

Reply:
I see the merit in this but it's going to take some getting used to for me 😅 [updated](https://github.com/CodeForBoulder/upswyng/pull/84/commits/ba2c1e8baaf98184857f87596db045d706dd6440)